components/onboarding-screen/issue-step-edit.tsx

```diff
@@ -199,7 +185,7 @@ export function IssueStepEdit() {
 
   const listCredentialDefinitions = async () => {
     try {
-      setLoading(true)
+      // setLoading(true)
       const response:any = await apiClient.get("/credentials/definitions");
```

Comment:
when you replace this for react-query following the examples of characters or showcase, you will be able to access isLoading from that transaction request and the types